### ISSUE TITLE
[Improvement] Tool Library should detect dead installation and Project Page links and mark them as "bad"

### ISSUE THREAD

**Issue by skfd** [MscrmTools/XrmToolBox#1301]
**Is your feature request related to a problem? Please describe.**

There are tools that can not be installed because the web site is dead. This should be caught by Tool Library.

Thank you!

**Comment by MscrmTools** [comment]
Any example?

**Comment by skfd** [comment]
Makes sense, but this is just an example. There might be other tools with broken links. I would say it would be nice for website\too library to periodically check the links. The same way wikipedia does dead link detection...

**Comment by MscrmTools** [comment]
Sure but wrong project url does not mean the nuget package is not available anymore so what should I do? remove them anyway? I'm not sure on the direction to take on this one

**Comment by skfd** [comment]
I would say just showing "⚠️Currently not available" label would be enough. Right now trying to install just does nothing -- it is basically an unhandled behavior, it should explicitly say that tool can not be installed because of the problem on developer's side.
